plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/plugin.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?eclipse version="3.4"?>
+<plugin>
+
+  <!-- temporary parent page -->
+  <!-- To be moved to a more general bundle
+       https://github.com/GoogleCloudPlatform/gcloud-eclipse-tools/issues/507 -->
+  <extension point="org.eclipse.ui.propertyPages">
+    <page
+        id="com.google.cloud.tools.eclipse"
+        name="Google"
+        class="com.google.cloud.tools.eclipse.properties.GooglePropertyPage">
+    </page>
+  </extension>
+
+  <extension point="org.eclipse.ui.propertyPages">
+    <page
+        id="com.google.cloud.tools.eclipse.appengine.deploy"
+        name="App Engine Deployment"
+        category="com.google.cloud.tools.eclipse"
+        class="com.google.cloud.tools.eclipse.appengine.deploy.ui.DeployPropertyPage">
+      <enabledWhen> <!-- this will probably be too strict when App Engine Flex becomes supported -->
```

Comment:
We should [define some of our own standard expressions](common=split) that can be referenced here.  I think it should be safe to not use `forcePluginActivation` since the facets bundle should be started if we're dealing with faceted projects.